pinc/User.inc

```diff
@@ -119,7 +119,7 @@ class User
                         // OK if we can't.
                         if(!header_sent())
                         {
-                            setcookie("profile");
+                            dp_setcookie("profile");
```

Comment:
This is going to fail because the `dp_setcookie()` definition requires a value. I guess in that function declaration we should default `$value=''`.